src/pages/hoyoungSohn/Login/Login.js

```diff
@@ -1,9 +1,81 @@
 import React from 'react';
+import { Link } from 'react-router-dom';
 import './Login.scss';
 
 class LoginHoYoung extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      email: '',
+      password: '',
+    };
+  }
+
+  goToMain = () => {
+    this.props.history.push('/main-young');
+    // fetch('http://10.58.3.77:8000/user_login/', {
+    //   method: 'POST',
+    //   body: JSON.stringify({
+    //     email: this.state.email,
+    //     password: this.state.password,
+    //   }),
+    // })
+    //   .then(response => response.json())
+    //   .then(result => console.log('결과: ', result));
+  };
+
+  handleIdInput = e => {
+    this.setState({
+      email: e.target.value,
+    });
+  };
+
+  handlePwInput = e => {
+    this.setState({
+      password: e.target.value,
+    });
+  };
```

Comment:
계산된 속성명을 활용해 다음과 같이 작성할 수 있습니다.

```suggestion
  handleInput = e => {
    this.setState({
      [e.target.name]: e.target.value,
    });
 };
```

대신 각각의 input 태그에 name attribute를 각 state의 키 값에 맞게 입력해주셔야 합니다.